### DISCUSSION TITLE
How many emails is Postal SMTP able send per second or per minute?

### DISCUSSION THREAD

**Discussion by didoMova** [postalserver/postal#1835] [Help with using Postal]
How many emails is Postal SMTP able to send per second or per minute? 

The reason why I'm asking is that I have a huge mail list and want to send it in the shortest period possible.

Do you know how to hit the limit with Postal?  

My VPS is: 4 vCPU Cores & 8 GB RAM

**Comment by willpower232** [comment]
With Postal v2 you can scale up the worker containers to increase your speed, there isn't a specific amount that could or could not be sent as there are too many variable to say precisely.

**Comment by didoMova** [comment]
Thanks for your help. Any tips you can suggest to check before my first try?

**Reply by willpower232** [reply]
the two biggest limits will be worker amount and MySQL configuration so it might be worth searching for that and configuring before you get started too much

**Reply by didoMova** [reply]
I'll look at MySQL configuration definitely, thank you.

Just don't understand what mean exactly worker amount?

**Reply by willpower232** [reply]
> With Postal v2 you can scale up the worker containers to increase your speed

**Reply by didoMova** [reply]
inside Postal dashboard or? 
Sorry, can't find it by myself.
Appreciate your help :)
